### PR TITLE
720 decrease the size of the buildlog module standardization

### PR DESCRIPTION
**Pull request by benedeki**
* Adhere to Scalastyle
* Follow some best-practices:
** typed implicits
** types declared for public members
* setup logging for tests - style, filtering
* remove direct output to console (println, Dataframe.show, etc)
* Added missing License

### CODE REVIEW

File: standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/DateTimeSuite.scala [hunk 1]
```diff
@@ -64,8 +64,7 @@ class DateTimeSuite extends FunSuite with SparkTestBase {
 
   test("Date Time Standardization Example with fixed schema should work") {
     val std = StandardizationInterpreter.standardize(data, schemaOk, "dates")
-    std.show(false)
-    std.printSchema
-    println(std.schema.prettyJson)
+    logDataFrameContent(std)
+    //TODO finish test
```
Comment (Zejnilovic) [inline standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/DateTimeSuite.scala]: What is this TODO?

Comment (Zejnilovic) [reply standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/DateTimeSuite.scala]: Cool

File: standardization/src/test/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Suppress validation error logs from SchemaChecker
+log4j.appender.forschemachecker=org.apache.log4j.ConsoleAppender
+log4j.appender.forschemachecker.target=System.err
+log4j.appender.forschemachecker.layout=org.apache.log4j.PatternLayout
+log4j.appender.forschemachecker.layout.ConversionPattern=[%p] %d{yy/MM/dd HH:mm:ss} %c{1}: %m%n
+log4j.appender.forschemachecker.filter.01=org.apache.log4j.varia.StringMatchFilter
+log4j.appender.forschemachecker.filter.01.StringToMatch=Validation error for column
+log4j.appender.forschemachecker.filter.01.AcceptOnMatch=false
+log4j.logger.za.co.absa.enceladus.standardization.interpreter.stages.SchemaChecker$=ERROR, forschemachecker
```
Comment (GeorgiChochov) [inline standardization/src/test/resources/log4j.properties]: This is probably the only module where this makes sense.

File: standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/CounterPartySuite.scala [hunk 3]
```diff
@@ -24,21 +24,21 @@ import za.co.absa.enceladus.utils.error.ErrorMessage
 case class Root(ConformedParty: Party, errCol: Seq[ErrorMessage] = Seq.empty)
 case class Party(key: Integer, clientKeys1: Seq[String], clientKeys2: Seq[String])
 
-class CounterPartySuite extends FunSuite with SparkTestBase {
+class CounterPartySuite extends FunSuite with SparkTestBase with LoggerTestBase{
```
Comment (GeorgiChochov) [inline standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/CounterPartySuite.scala]:
```suggestion
class CounterPartySuite extends FunSuite with SparkTestBase with LoggerTestBase {
```

**Pull request merged**